MewsSystems/mews-flutter
# feat: [DX-295] Update font

#### Summary

- Added missing font-weight
- Added missing slashed zero feature
<details><summary>Screenshots</summary>

`FontWeight.w500` was missing

![CleanShot 2023-10-11 at 18 20 47](https://github.com/MewsSystems/mews-flutter/assets/9210422/88f11a9e-5772-4252-bb41-780aaaec6061)

All font weights

![CleanShot 2023-10-11 at 18 22 52](https://github.com/MewsSystems/mews-flutter/assets/9210422/40d1ba22-1812-4568-8ff6-dde6dc6fada7)

Just the font weights that are in use in `optimus`

![CleanShot 2023-10-11 at 18 31 18](https://github.com/MewsSystems/mews-flutter/assets/9210422/b0ff2bc5-7be2-48f2-af2a-5741dcbc8608)


</details>


#### Testing steps

- Open any typography story
- Check if font, font size, and font-weight are displayed correctly

#### Follow-up issues

None

#### Check during review

- Verify against Jira issue.
- Is the PR over 300 additions? Consider rejecting it with advice to split it. Is it over 500 additions? It should definitely be rejected.
- Unused code removed.
- Build passing.
- Is it a bug fix? Check that it is covered by a proper test (unit or integration).
